source/items/tagtool.md

```diff
@@ -0,0 +1,69 @@
+
+2. While you are holding the tag tool, you will see all existing tags overlaying their blocks in-world, and they will be highlighted with a white border.
+
+3. Right-click the tag tool in the air to open its GUI. This allows you to type in the tag that you want to apply or remove.  Close the GUI afterwards. The GUI also shows a list of all currently tagged blocks.
+
+4. Left-click on a block to apply the current tag to that block. Left-click again to remove the tag.
+
+    * It is possible (although unusual) for a single block to have multiple tags. To remove a tag it must be the current tag entered in the GUI.
+
+5. When finished, use the [scan tool](../../source/items/scantool) to save the schematic, including the tags. Tags will only take effect if the anchor block for the scan is the same anchor used to store the tags. (When there is only one anchor block in the scan area, it will be selected automatically; if you are including multiple hut blocks in a single schematic then you may need to select the right one manually.)
+
+## Tag Anchor Block
+
+<div class="infobox box text-center">
+    <p style="text-align:center;"><img src="../../assets/images/items/tagsubstitutionblock.png" alt="Tag Anchor Block"></p>
+</div>
+<br>
+
+The Tag Anchor Block is also a creative-only item, whose sole purpose is to act as the tag-containing anchor for non-upgradeable decorations. There should typically be at most one of these in any given schematic. (It is possible to include more than one, but only one can be the actual anchor, and that's the only one that matters.)
+
+When built by a [Builder](../../source/workers/builder) or pasted normally, the Tag Anchor Block will disappear. As such it is only useful for tags that affect the build itself (but no other tags are interesting for decorations anyway). When pasting the schematic for editing, ensure that the left paste option is used to retain the block (similar to other [placeholder blocks](../../source/items/placeholderblocks)).
+
+## Current Tags
+
+The following tags are currently implemented (more may be added from time to time):
+
+* `groundlevel`
+
+    This tag is supported by all huts and decorations. It is optional. When included, it should only appear on exactly one block in the schematic that is at the nominal "ground level" of the building -- i.e. any blocks below this should be an underground "basement" area, blocks at the same level should be dirt/grass/other ground blocks (often [solid placeholders](../../source/items/placeholderblocks) are used for this, while underground can be either solid or light placeholders outside of the intended basement area), and blocks above should be the actual building. (If it is specified on more than one block, then only one will take effect. This is harmless if they're all on the same y level but can be confusing if not; hence the recommendation to stick with one at most.)

```

Comment:
tabs aren't needed at the beginning of paragraphs

Reply:
It is, when you want the paragraph to be a child of the bullet point and not separate.  At least if this is following normal markdown formatting rules.